handlers/common.go

```diff
@@ -53,7 +53,12 @@ func GetHandlers() (map[string]revents.EventHandler, error) {
 func logRequest(f revents.EventHandler) revents.EventHandler {
 	return func(event *revents.Event, cli *client.RancherClient) error {
 		logrus.Infof("Received event: Name: %s, Event Id: %s, Resource Id: %s", event.Name, event.ID, event.ResourceID)
-		return f(event, cli)
+		startTime := time.Now()
+		err := f(event, cli)
+		defer func() {
+			logrus.Infof("Name: %s, Event Id: %s, Resource Id: %s, Time Spent %v seconds on processing the event", event.Name, event.ID, event.ResourceID, time.Now().Sub(startTime).Seconds())
```

Comment:
`"Time Spent 1 seconds on processing the event"` doesn't really sound right. How about:
`Name: foo, Event Id: 6, Resource Id: 1c1, Process duration: .4 seconds`

Also, I think we should pick a precision other than the full 9 or 10 places. Maybe a precision of 4?
`Process duration: %.4f seconds`